doc/src/site/sphinx/deployment/aws_emr_edge_node.rst

```diff
@@ -2,7 +2,7 @@ Use Sparkling Water with Amazon EMR from the Edge Node
 ------------------------------------------------------
 
 Sparkling Water can be deployed on a cluster on EMR or directly on EC2 machines. The jobs can be submitted
-from the edge node which has several security and other benefits.
+from the edge node which have several security and other benefits.
```

Comment:
why `have`?

Reply:
Thanks, I connected has with jobs, but the subject is the deployment